Ginger/GingerCoreCommon/Repository/ApplicationModelLib/POMModelLib/ApplicationPOMModel.cs

```diff
@@ -52,12 +52,46 @@ public ApplicationPOMModel()
             }
         }
 
+
+
+
         [IsSerializedForLocalRepository]
         public ObservableList<ElementInfo> UnMappedUIElements = new ObservableList<ElementInfo>();
 
         [IsSerializedForLocalRepository]
         public ObservableList<ElementInfo> MappedUIElements = new ObservableList<ElementInfo>();
 
+
+        public enum eElementGroup
+        {
+            Mapped,
+            Unmapped
+        }
+
+
+        public ObservableList<string> ElementGroupList
+        {
+            get
```

Comment:
not sure why this ElementGroupList is needed, i guess you need it for Filter/grid cell but you can create it from eElementGroup enum when/where needed no? why to maintain 2 lists of the same? for each new enum which will be added we will need to update also ElementGroupList

Reply:
yea, it been used for the grid but i do not use it anymore,
Removed.